docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,30 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+
+.. _splash-with-timeout:
+
+splash:with_timeout
+-------------------
+
+Run the callback with the allowed timeout
+
+**Signature:** ``ok, result = splash:with_timeout(callback, timeout)``
```

Comment:
It's better for the eyes to have timeout first, so that you don't have to squint looking for the number somewhere after the callback definition. It would also match the Lua de-facto style convention neatly with 
```
with_callback(0.1, function()
   do_smth()
end)
```

Is there a precedent in non-Lua world calling for placing the timeout first?